gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
+from gammapy.image import make_empty_image
+
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import astropy.units as u
+from astropy.units.quantity import Quantity
+
+import numpy as np
+from scipy.integrate import quad
+
+def radial_gaussian_2D(r, sig):
+    """Radially symmetric Gaussian function for emulating acceptance curve.
+
+    Please give parameters in radians.
+
+    TODO: I can't define parameters as `~astropy.coordinates.Angle` because it conflicts with the integral call afterwards!!!
+
+    Parameters
+    ----------
+    r : `~numpy.ndarray`
+    Radial coordinate.
+    sig : float
+    Gaussian width.
+
+    Returns
+    -------
+    val: `~numpy.ndarray`
+    Gaussian evaluated at the requested radii.
+    """
+    ##norm = 1./(sig * np.sqrt(2. * np.pi))
+    ##norm *= 2. #due to definition only for r>=0
+    ##norm /= (2.*np.pi) #due to radial symmetry
+    norm = 1.
+    val = norm * np.exp(-np.power(r, 2.) / (2. * np.power(sig, 2.)))
+    return val
+
+
+def test_fill_acceptance_image():
+
+    #create empty image
+    image = make_empty_image()
+
+    #define center coordinate of the image
+    #in FITS the reference pixel is given by CRPIXn, and its coordinate by CRVALn
+    #the coordinate increment along the axis is given by CDELTn
+    x = image.header['CRVAL1']
+    y = image.header['CRVAL2']
+
+    #sanity checks: am I using galactic coordinates in degrees?
+    assert image.header['CTYPE1'] == 'GLON-CAR', "Error: x coordinate is not in Galactic coordinates!"
+    assert image.header['CTYPE2'] == 'GLAT-CAR', "Error: y coordinate is not in Galactic coordinates!"
+    assert image.header['CUNIT1'] == 'deg', "Error: x coordinate is not in degrees!"
+    assert image.header['CUNIT2'] == 'deg', "Error: y coordinate is not in degrees!"
+
+    center = SkyCoord(l=x*u.degree, b=y*u.degree, frame='galactic')
+
+    #define pixel sizes
+    x_pix_size = Angle(abs(image.header['CDELT1'])*u.degree)
+    y_pix_size = Angle(abs(image.header['CDELT2'])*u.degree)
+
+    #define radial acceptance and offset angles
+    offset = Angle(np.arange(0., 30., 0.1), unit=u.degree)
+    acceptance = np.zeros_like(offset)
+    sigma = Angle(1.0, unit=u.degree) #gaussian width
```

Comment:
I usually write
```python
a = Angle(42, 'degree')
```
instead of
```python
a = Angle(42 * u.degree)
```
If you don't mind, could you please also change to write Quantities by putting the units as strings?

There's two small advantages:
1) I find "degree" more readable than "u.degree"
2) Pycharm and other Python IDEs do static code analysis. But all the symbols in `astropy.units` are dynamically generated on import, so I get a warning displayed for a missing symbol "u.degree", which is distracting, because really all is OK.